examples/cccbenchplushwloc.cpp

```diff
@@ -0,0 +1,58 @@
+#include <iostream>
+#include <filesystem>
+
+#include "sys-sage.hpp"
+#include "parsers/cccbench.hpp"
+
+void usage(char* argv0)
+{
+    std::cerr << "usage: " << argv0 << " <hwloc xml path> <cccbench csv path>" << std::endl;
+    return;
+}
+
+int main(int argc, char *argv[])
+{
+    string topoPath;
+    const char *cccPath;
+    if(argc == 3){
+        topoPath = argv[1];
+        cccPath = argv[2];
+    }
+    else{
+        usage(argv[0]);
+        return 1;
+    }
+
+    //create root Topology and one node
+    Topology* topo = new Topology();
+    Node* n = new Node(topo, 1);
+
+    if(parseHwlocOutput(n, topoPath) != 0) { //adds topo to a next node
+        usage(argv[0]);
+        return 1;
+    }
+    cout << "-- End parseHwlocOutput" << endl;
+
+    cout << "Total num HW threads: " << topo->GetNumThreads() << endl;
+
+    cout << "---------------- Printing the whole tree ----------------" << endl;
+    topo->PrintSubtree(2);
+    cout << "----------------                     ----------------" << endl;
+    
+    
+    auto cccparser = new CccbenchParser(cccPath);
```

Comment:
Could you please also create a single function, such as parseCccbenchOutput(n, cccPath), in the cccbench.cpp/hpp parser for this, so that it works analogously to the parseHwlocOutput?
The function could basically do something like:
```
auto cccparser = new CccbenchParser(cccPath);
cccparser->applyDataPaths(n);
delete cccparser;
return;
```